modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/JdbcSqlStreamingTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.query;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Properties;
+import java.util.UUID;
+import org.apache.ignite.IgniteJdbcThinDriver;
+import org.apache.ignite.internal.IgniteEx;
+import org.apache.ignite.internal.processors.cache.index.AbstractIndexingCommonTest;
+import org.junit.Test;
+
+/**
+ *
+ */
+public class JdbcSqlStreamingTest extends AbstractIndexingCommonTest {
+    /** */
+    private static final int ROW_NUM = 1600;
+
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testStreamingWithFunctionValues() throws Exception {
```

Comment:
Can you, please, rewrite this test using Ignite code style?

https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines

Reply:
Done.